### PR TITLE
Feature: Cart

### PR DESCRIPTION
**Pull request by jason-villa-accenture**
Modified:
src\routes\index.jsx
src\constants\routes\index.js

Added:
src\pages\user\Cart.jsx
src\components\cart
src\components\cart\CartCard.jsx
src\components\cart\CartSummary.jsx
src\components\cart\index.js

### CODE REVIEW

File: src/components/cart/CartCard.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Checkbox, Typography, Button } from 'antd';
+import { FaPlus, FaMinus } from 'react-icons/fa';
+import PRODUCT_IMAGE from '../../assets/images/product_1.webp';
```
Comment (AccentureJace) [inline src/components/cart/CartCard.jsx]: utilize index.js

File: src/constants/routes/index.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 export const PRODUCT_DASHBOARD = '/';
 export const PRODUCT_DETAILS = '/product/:id';
+export const CART = '/cart';
```
Comment (AccentureJace) [inline src/constants/routes/index.js]: this should be PATH_CART

File: src/pages/user/Cart.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { CartCard, CartSummary } from '../../components/cart';
```
Comment (AccentureJace) [inline src/pages/user/Cart.jsx]: follow import ordering

File: src/routes/index.jsx [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import { createBrowserRouter } from 'react-router-dom';
 import Dashboard from '../pages/user/Dashboard';
 import ProductDetails from '../pages/user/ProductDetails';
+import Cart from '../pages/user/Cart';
```
Comment (AccentureJace) [inline src/routes/index.jsx]: utilize index.js

File: src/components/cart/CartCard.jsx
Comment (AccentureJace) [inline src/components/cart/CartCard.jsx:1]: create index.js file inside cart folder.
import also the files in the component's index.js. refer to apps and products index.js and components' index.js

**Review by AccentureJace — CHANGES_REQUESTED** [review]
Please see request changes

File: src/components/cart/CartCard.jsx (hunk 1 above)
Comment (jason-villa-accenture) [reply src/components/cart/CartCard.jsx]: done.

File: src/constants/routes/index.js (hunk 2 above)
Comment (jason-villa-accenture) [reply src/constants/routes/index.js]: done.

File: src/pages/user/Cart.jsx (hunk 3 above)
Comment (jason-villa-accenture) [reply src/pages/user/Cart.jsx]: done

File: src/routes/index.jsx (hunk 4 above)
Comment (jason-villa-accenture) [reply src/routes/index.jsx]: done.

File: src/components/cart/CartCard.jsx
Comment (jason-villa-accenture) [reply src/components/cart/CartCard.jsx:1]: done

**Review by AccentureJace — APPROVED** [review]
LGTM

**Review by AccentureAngelo — APPROVED** [review]

**Comment by AccentureJace** [comment]
Closing this PR since this is no longer applicable.

**Pull request closed without merging**